plots/measure.js

```diff
@@ -31,9 +59,29 @@ const ChromeLauncher = require('../chrome-launcher/chrome-launcher.js');
 const Printer = require('../lighthouse-cli/printer');
 const assetSaver = require('../lighthouse-core/lib/asset-saver.js');
 
-const NUMBER_OF_RUNS = 20;
+const DISABLE_DEVICE_EMULATION = args['disable-device-emulation'];
+const DISABLE_CPU_THROTTLING = args['disable-cpu-throttling'];
+const DISABLE_NETWORK_THROTTLING = args['disable-network-throttling'];
+const REUSE_CHROME = args['reuse-chrome'];
+const KEEP_FIRST_RUN = args['keep-first-run'] || !REUSE_CHROME;
+const SITES_PATH = args['sites-path'];
+const SUBSET = args['subset'];
+const SITE = args['site'];
+const CUSTOM_NUMBER_OF_RUNS = args['n'];
 
-const URLS = [
+// Running it n + 1 times if the first run is deliberately ignored
+// because it has different perf characteristics from subsequent runs
+// (e.g. DNS cache which can't be easily reset between runs)
+const NUMBER_OF_RUNS = (CUSTOM_NUMBER_OF_RUNS || 20) + (KEEP_FIRST_RUN ? 0 : 1);
```

Comment:
it's unexpected that number_of_runs == 1 will provide two runs for each site. Can we make sure the number here will match # of actual runs done?